Source/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -901,6 +901,22 @@ - (void)setAccessibilityLabel:(NSString *)accessibilityLabel
 {
   _bridge_prologue_write;
   _setAccessibilityToViewAndProperty(_accessibilityLabel, accessibilityLabel, accessibilityLabel, accessibilityLabel);
+  if (@available(iOS 11.0, *)) {
+    _setAccessibilityToViewAndProperty(_accessibilityAttributedLabel, [[NSAttributedString alloc] initWithString:accessibilityLabel], accessibilityAttributedLabel, [[NSAttributedString alloc] initWithString:accessibilityLabel]);
+  }
+}
+
+- (NSAttributedString *)accessibilityAttributedLabel
+{
+  _bridge_prologue_read;
+  return _getAccessibilityFromViewOrProperty(_accessibilityAttributedLabel, accessibilityAttributedLabel);
+}
+
+- (void)setAccessibilityAttributedLabel:(NSAttributedString *)accessibilityAttributedLabel
+{
+  _bridge_prologue_write;
+  { _setAccessibilityToViewAndProperty(_accessibilityAttributedLabel, accessibilityAttributedLabel, accessibilityAttributedLabel, accessibilityAttributedLabel); }
+  { _setAccessibilityToViewAndProperty(_accessibilityLabel, accessibilityAttributedLabel.string, accessibilityLabel, accessibilityAttributedLabel.string); }
```

Comment:
This looks a bit strange. Are attributed labels supposed to overwrite non-attributed ones? Since these attributed labels are still in beta, I guess we don't know yet?

Reply:
This is in fact the behaviour I'm observing. It's like UILabel's `attributedText` and `text` attributes, which override themselves (confirmed in Playground with XC9 GM).

Reply:
Cool by me then :)